README.md

```diff
@@ -1,6 +1,14 @@
 # GatherPress
 
-> GatherPress, powering our community's event management needs.
+Stable tag: 0.29.0-alpha  
+Tested up to: 6.5.2  
+License: GPL v2 or later  
+Tags: events, event, meetup, community  
+Contributors: mauteri, hrmervin, jmarx, meaganhanes, pbrocks  
```

Comment:
Let's change this list to same as credits. Just people listed as leads and team under Settings -> Credits (see other comment below).

Reply:
> I would like to see my handle in the list, too.

Now that I look at `/wp-admin/edit.php?post_type=gatherpress_event&page=gatherpress_credits` you can please ignore my first comment @mauteri .

And a big **thank you** for having me in the team! :man_dancing:  _May I not be your xz contributor._

CC: @patriciabt

Reply:
Looks good!